Containerfile

```diff
@@ -1,19 +1,14 @@
 # SPDX-License-Identifier: Apache-2.0
-
```

Comment:
Do we need to name this file Dockerfile? Look at 'git log -- Dockerfile Containerfile' to see a sordid history of different naming approaches we've had over the past (I think we flipped to Dockerfile when gitlab CI wouldn't recognize Containerfile, and know we accidentally flipped back when overhauling to use kubebuilder).  Of course, a rename would be wise as an independent commit.

Reply:
It doesn't matter to me either way, for Openshift CI we specify the Dockerfile path so in this case I would just specify `Containerfile`  
example: https://github.com/openshift/release/pull/59746/files#diff-fc2f8ca9e802b37cde660774cd4d43ed044d7d2365ead0ee4f242016f7dcaef2R13